srv/apns/push_service.go

```diff
@@ -140,14 +185,11 @@ func (p *pushService) BuildDeliveryPointFromMap(kv map[string]string, dp *push.D
 	return nil
 }
 
-func NewBinaryPushService() *pushService {
-	return newPushService(binary_api.NewRequestProcessor(maxNrConn))
-}
-
-func newPushService(requestProcessor common.PushRequestProcessor) *pushService {
+func NewPushService() *pushService {
 	return &pushService{
-		requestProcessor: requestProcessor,
-		nextMessageId:    1,
+		binaryRequestProcessor: binary_api.NewRequestProcessor(maxNrConn),
```

Comment:
makes sense, would need both if different PSPs have different preferences of p8 vs binary api cert